client/my-sites/stats/site.jsx

```diff
@@ -524,6 +533,60 @@ function StatsBody( { siteId, chartTab = 'views', date, context, isInternal, ...
 	// TODO: Fix isOdysseyStats to include the environment running on WP-Admin of Simple sites.
 	const isRunningOnWPAdmin = document.getElementById( 'wpadminbar' );
 
+	const statsAdminVersion = useSelector( ( state ) =>
+		getJetpackStatsAdminVersion( state, siteId )
+	);
+
+	const gatedTrafficPage = useSelector( ( state ) => {
```

Comment:
Could we reuse [the gate](https://github.com/Automattic/wp-calypso/blob/f2a4ecb4a38cf741bcbb0cd3020fc6d96ebe5ed2/client/my-sites/stats/site.jsx#L254-L258) for the Traffic page?